src/xenia/kernel/xam/content_manager.cc

```diff
@@ -69,14 +71,14 @@ std::filesystem::path ContentManager::ResolvePackageRoot(
   auto content_type_str = fmt::format("{:08X}", uint32_t(content_type));
 
   // Package root path:
-  // content_root/title_id/content_type/
-  return root_path_ / title_id_str / content_type_str;
+  // content/user_id/title_id/content_type
+  return root_path_ / cvars::user_profile / title_id_str / content_type_str;
```

Comment:
In the online fork I had, a few people liked changing their name a few times.

While I didn't touch content files, [I did have data storage for account files](https://github.com/Bo98/xenia-burnout5/commit/a4033c724077955be4a5f02f9558743fff2bead2#diff-46230285d4100d8049d651c5e77516c7a53cbbef8e0e33578dd738479a89dc51R197-R200) and I used (offline) XUID for those.

Changing names should be save game compatible. Changing XUIDs is often not, so is more stable.